core/jvm/src/main/scala/cats/effect/IOApp.scala

```diff
@@ -25,46 +25,29 @@ trait IOApp {
 
   def run(args: List[String]): IO[Int]
 
+  protected def platform: unsafe.IOPlatform = unsafe.IOPlatform.Globals.platform
```

Comment:
If I wanted to override *just* the `ExecutionContext`, would I have to define my own scheduler? Seems like there isn't a particularly easy way to access the default one since `GlobalsPlatform` is declared `private[unsafe]`, or just override `executionContext`. 

maybe have a non-private, final `defaultPlatform` and expose `with`/`copy` methods on `IOPlatform` to override? or expose the global `scheduler` somehow. not sure which is cleaner

Reply:
actually, the override might not be a good idea, since it would allocate two `executioncontexts` probably?

Reply:
Yeah, with this encoding, any access of Globals object causes the default pools to be instantiated. We could put some default factory methods on the `IORuntime` companion though.

Reply:
Is it possible to access `IORuntime.Globals.scheduler` from an `IOApp`? The `protected` modifier seems to suggest not (forgive my laziness in not cloning your PR)

Reply:
OK just pushed an update that supports partial reuse of default EC/Scheduler.

Reply:
> Is it possible to access `IORuntime.Globals.scheduler` from an `IOApp`? The `protected` modifier seems to suggest not (forgive my laziness in not cloning your PR)

As of latest push, you can use `IORuntime.createDefaultScheduler()`